docs/index.md

```diff
@@ -1,17 +1,30 @@
----
+--------------------------------------------------------------------------------
+
 layout: home
-nav_order: 1
----
+
+## nav_order: 1
```

Comment:
This is also incorrect, probably caused by mdformat as well

Reply:
Also fixed in d9ee501